index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nelson Araujo's Portfolio</title>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    <header>
+        <section class="mobile-nav">
+            <div class="landing-header">
+                <div class="header-image"></div>
+                <button id='open-menu-btn' class="menu-btn">
+                    <img src="./assets/ic_menu.svg">
+                </button>
+            </div>
+            <nav id='menu-container'>
+                <div class="top-image"></div>
+                <button id='close-menu-btn' class="menu-btn">
+                    <img src="./assets/Union.svg">
+                </button>
+                <ul>
+                    <li>
+                        <a>Hello</a>
+                    </li>
+                    <li>
+                        <a>Portfolio</a>
+                    </li>
+                    <li>
+                        <a>About</a>
+                    </li>
+                    <li>
+                        <a>Contact</a>
+                    </li>
+                </ul>
+                <div class="bottom-image"></div>
+            </nav>
+        </section>
+    </header>
+    <main>
+        <section class="introduction-mobile">
+            <h1>Nelson Araujo</h1>
+            <p>Hello! and welcome to my website. I am a web developer and why not your future collaborator? Take some minutes to look at my works. And if you want to know more about me, do not hesitate contacting.</p>
+            <button class="main-btn">
+                Start collaborating
+            </button>
+        </section>
+        <div class="deco-image"></div>
```

Comment:
To stay in sync with the template, I recommend making this image a full-width one according to the design

Your design
![std pic](https://user-images.githubusercontent.com/84930355/204900011-acaa9db5-c7f4-42e7-a117-73394556ad0b.JPG)

Figma design
![fig width](https://user-images.githubusercontent.com/84930355/204900051-531ae094-32ee-48c5-8e6c-d60222574f1a.JPG)

Reply:
Hello @DuaneDave, thanks for the advices, the image is actually set as full-width, but will change the background-size property from 'contain' to 'cover'.